test/acceptance/authz/rbac_auto_no_permissions_test.go

```diff
@@ -125,7 +125,7 @@ func TestAuthzAllEndpointsNoPermissionDynamically(t *testing.T) {
 			if slices.Contains(ignoreEndpoints, endpoint.path) {
 				return
 			}
-
+			t.Log(resp.Body)
```

Comment:
debug leftover? :)

Reply:
🤦‍♂️ i thought i removed, good catch on it